surveys/serializer_field_generators.py

```diff
@@ -4,6 +4,31 @@
 from .generators import BaseFieldGenerator as BaseBaseFieldGenerator
 
 
+class BlankIntegerField(serializers.IntegerField):
```

Comment:
You can maybe try to set `initial=None` to the field see http://www.django-rest-framework.org/api-guide/fields/#initial